### ISSUE TITLE
Router.current().params return nothing

### ISSUE THREAD

**Issue by AndreyAntipov** [iron-meteor/iron-router#1416]
Hello. 
I don't understand, why ```Router.current().params``` return empty object, is that correct behaviour with route below?

__Url:__
[localhost:3000/dashboard/objects/33123123/edit/](#)

__Route:__
```coffeescript
Router.route 'edit',
  {
    path: '/dashboard/objects/:_id/edit'
    waitOn: ->
      Meteor.subscribe('images')
      Meteor.subscribe('zones')
      Meteor.subscribe('objects')
    data: ->
      {
        _magic: @params._id
      }
  }
```

__Returns:__

``` console.log(Router.current().params); ```
Output: ```empty object```

``` console.log(this._magic); ```
Output: ```33123123```

**Comment by gibex** [comment]
Seems like you can't access the params Object directly, 
use Router.current().params._id

**Comment by tgoorden** [comment]
Check this (closed) issue: #1318

**Comment by RubenGarica1** [comment]
Router.current().getParams()
